solara/website/pages/documentation/examples/__init__.py

```diff
@@ -37,6 +41,11 @@ def Layout(children):
             with solara.Column(style={"max-width": "min(100%, 1024px)", "width": "100%"}):
                 if route_current.path != "/":
                     solara.Button("View source code on GitHub", icon_name="mdi-github-circle", href=github_url, class_="ma-2", target="_blank", text=True)
+                if route_current.path in pycafe_projects:
+                    pycafe_url = f"https://py.cafe/solara/{route_current.path}"
+                    solara.Button(
+                        "Run this example on PyCafe", icon_name="mdi-coffee-to-go-outline", href=pycafe_url, class_="ma-2", target="_blank", text=True
```

Comment:
Not really central to the content, but maybe emphasising the editability would be a good marketing move
 
```suggestion
                        "Run and edit on PyCafe", icon_name="mdi-coffee-to-go-outline", href=pycafe_url, class_="ma-2", target="_blank", text=True
```